library/ssl_tls.c

```diff
@@ -5657,6 +5660,8 @@ static unsigned char ssl_serialized_session_header[] = {
     ( SSL_SERIALIZED_SESSION_CONFIG_BITFLAG >> 0 ) & 0xFF,
 };
 
+#if defined(MBEDTLS_SSL_SESSION_TICKETS) || ( defined(MBEDTLS_SSL_NEW_SESSION_TICKET) && defined(MBEDTLS_SSL_SRV_C) )
+
 /*
```

Comment:
Similar to the comments in `ssl.h`, this macro seems mismatch with what is in the header.  And it also leaves `ssl_serialized_session_header` unused.